tests/sqllogictest.rs

```diff
@@ -67,7 +70,10 @@ fn test_disk(name: &str) {
     ));
     let mut tester = sqllogictest::Runner::new(DatabaseWrapper { rt, db });
     tester.enable_testdir();
-    tester.run_file(Path::new("tests/sql").join(name)).unwrap();
+    tester
+        .run_file(Path::new("tests/sql").join(name))
+        .map_err(|e| panic!("{}", e))
```

Comment:
... why panic before unwrap?

Reply:
To print the error in Display format, which is more readable than Debug format...